Kitodo/src/main/java/de/sub/goobi/forms/IndexingForm.java

```diff
@@ -406,14 +407,12 @@ private void resetGlobalProgress() {
         }
     }
 
-    private static String readMapping() throws ParseException {
-        JSONParser parser = new JSONParser();
+    private static String readMapping() {
         ClassLoader classloader = Thread.currentThread().getContextClassLoader();
         try (InputStream inputStream = classloader.getResourceAsStream("mapping.json")) {
             String mapping = IOUtils.toString(inputStream, "UTF-8");
-            Object object = parser.parse(mapping);
-            JSONObject jsonObject = (JSONObject) object;
-            return jsonObject.toJSONString();
+            JsonObject jsonObject = Json.createReader(new StringReader(mapping)).readObject();
```

Comment:
Same here. `JsonReader` is closeable so using a try-with-resources statement for closing the object is maybe a good idea.